keyboards/whtkbrd/keymaps/default/keymap.c

```diff
@@ -0,0 +1,123 @@
+    ),
+    [LAYER_BRA_SYM] = LAYOUT(
+        XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,                      XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,
+        XXXXXXX,       XXXXXXX,       XXXXXXX,       KC_LCBR,       KC_RCBR,       XXXXXXX,                      KC_DQUO,       KC_COMM,       KC_QUES,       KC_COLN,       XXXXXXX,       XXXXXXX,
+        XXXXXXX,       XXXXXXX,       KC_LT,         KC_LPRN,       KC_RPRN,       KC_GT,                        KC_QUOT,       KC_DOT,        KC_EXLM,       KC_SCLN,       XXXXXXX,       XXXXXXX,
+        XXXXXXX,       _______,       XXXXXXX,       KC_LBRC,       KC_RBRC,       XXXXXXX,                      KC_AT,         KC_HASH,       KC_DLR,        KC_PERC,       XXXXXXX,       XXXXXXX,
+                                      XXXXXXX,       _______,       _______,       _______,                      _______,       _______,       _______,       XXXXXXX
+    ),
+    [LAYER_FUN_NUM] = LAYOUT(
+        XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,                      XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,       XXXXXXX,
+        XXXXXXX,       XXXXXXX,       KC_F1,         KC_F2,         KC_F3,         KC_F4,                        KC_0,          KC_1,          KC_2,          KC_3,          RESET,         XXXXXXX,
+        XXXXXXX,       XXXXXXX,       KC_F5,         KC_F6,         KC_F7,         KC_F8,                        KC_DOT,        KC_4,          KC_5,          KC_6,          XXXXXXX,       XXXXXXX,
+        XXXXXXX,       _______,       KC_F9,         KC_F10,        KC_F11,        KC_F12,                       KC_COMM,       KC_7,          KC_8,          KC_9,          XXXXXXX,       XXXXXXX,
+                                      XXXXXXX,       _______,       _______,       _______,                      _______,       _______,       _______,       XXXXXXX
+    ),
+};
+
+void update_leds(void) {
+    switch (get_highest_layer(layer_state)) {
+        case LAYER_ALPHA:
+            pwr_led_off();
+            stat_led_off();
+            break;
+        case LAYER_SYM_NAV:
+            pwr_led_off();
+            stat_led_on();
+            break;
+        case LAYER_BRA_SYM:
+            pwr_led_on();
+            stat_led_off();
+            break;
+        case LAYER_FUN_NUM:
+            pwr_led_on();
+            stat_led_on();
+            break;
+        case LAYER_THUMB:
+            break;
+        default:
+            break;
+    }
+}
+
+void post_process_record_user(uint16_t keycode, keyrecord_t *record) {
+    update_leds();
+}
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, LAYER_SYM_NAV, LAYER_BRA_SYM, LAYER_FUN_NUM);
+}
```

Comment:
This would be simpler, I think.
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
    state = update_tri_layer_state(state, LAYER_SYM_NAV, LAYER_BRA_SYM, LAYER_FUN_NUM);
    switch (get_highest_layer(state)) {
        case LAYER_ALPHA:
            pwr_led_off();
            stat_led_off();
            break;
        case LAYER_SYM_NAV:
            pwr_led_off();
            stat_led_on();
            break;
        case LAYER_BRA_SYM:
            pwr_led_on();
            stat_led_off();
            break;
        case LAYER_FUN_NUM:
            pwr_led_on();
            stat_led_on();
            break;
        case LAYER_THUMB:
            break;
        default:
            break;
    }
    return state;
}
```

Reply:
Nice! I took over your suggestion but left the `update_leds` as a separate function (now with the `state` as argument).